demos/serverless/README.md

```diff
@@ -40,7 +40,15 @@ npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting -u f
 ```
 
 #### Media Capture
-If you want to use media capture, an S3 bucket needs to be created for each region.
+You can use the live connector feature to broadcast the captured artifacts. The default layout is vertical layout and FHD resolution. You can use the playback url that showed up in any M3U8 player, such as `https://www.hlsplayer.net/`.
+If you want to try differnt layout configuration, please check our [ public document](https://docs.aws.amazon.com/chime-sdk/latest/APIReference/API_media-pipelines-chime_CreateMediaLiveConnectorPipeline.html).
+```
+cd demos/serverless
+npm install
+npm run deploy -- -r us-east-1 -b <deploy-bucekt> -i af-south-1,eu-south-1 -s <my-stack-name> -a meeting
```

Comment:
Are these necessary regions or just an example? Suggest keeping the defaults over specific if not needed.

Reply:
it's just an example. I will keep consistent with the existing examples.